tests/libcxx/expected_results.txt

```diff
@@ -566,48 +524,43 @@ std/utilities/allocator.adaptor/allocator.adaptor.members/construct_pair_rvalue.
 std/utilities/allocator.adaptor/allocator.adaptor.members/construct_pair_values.pass.cpp FAIL
 std/utilities/allocator.adaptor/allocator.adaptor.members/construct_type.pass.cpp FAIL
 
-# Tests ignore the return value of sto{meow}, triggering nodiscard warnings
-std/strings/string.conversions/stod.pass.cpp FAIL
-std/strings/string.conversions/stof.pass.cpp FAIL
-std/strings/string.conversions/stoi.pass.cpp FAIL
-std/strings/string.conversions/stol.pass.cpp FAIL
-std/strings/string.conversions/stold.pass.cpp FAIL
-std/strings/string.conversions/stoll.pass.cpp FAIL
-std/strings/string.conversions/stoul.pass.cpp FAIL
-std/strings/string.conversions/stoull.pass.cpp FAIL
-
 # We define __cpp_lib_has_unique_object_representations in C++17 mode; test error says it
 # "should not be defined when TEST_HAS_BUILTIN_IDENTIFIER(__has_unique_object_representations) || TEST_GCC_VER >= 700 is not defined!"
-std/language.support/support.limits/support.limits.general/type_traits.version.pass.cpp:1 FAIL
+std/language.support/support.limits/support.limits.general/type_traits.version.compile.pass.cpp FAIL

```

Comment:
Test error now says "__cpp_lib_reference_from_temporary should be defined in c++2b".

Reply:
Ditto features we don't implement yet, P2255R2 "Type Traits To Detect References Binding To Temporaries".